Test fails on Debian

https://github.com/containers/podman/blob/4c8ad635689cbc43a68d403874bbe3a6b66b7637/libpod/container_internal_linux_test.go#L29

This test fails on my Debian laptop with:

```
=== RUN   TestGenerateUserPasswdEntry
    container_internal_linux_test.go:29: 
        	Error Trace:	/<<PKGBUILDDIR>>/_output/src/github.com/containers/podman/libpod/container_internal_linux_test.go:29
        	Error:      	Not equal: 
        	            	expected: "_flatpak:*:123:456:Flatpak system-wide installation helper:/:/bin/sh\n"
        	            	actual  : "123:*:123:456:container user:/:/bin/sh\n"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,2 +1,2 @@
        	            	-_flatpak:*:123:456:Flatpak system-wide installation helper:/:/bin/sh
        	            	+123:*:123:456:container user:/:/bin/sh
        	            	 
        	Test:       	TestGenerateUserPasswdEntry
--- FAIL: TestGenerateUserPasswdEntry (0.00s)
```

--- a/libpod/container_internal_linux_test.go
+++ b/libpod/container_internal_linux_test.go
@@ -15,7 +15,7 @@ func TestGenerateUserPasswdEntry(t *testing.T) {
 		config: &ContainerConfig{
 			Spec: &spec.Spec{},
 			ContainerSecurityConfig: ContainerSecurityConfig{
-				User: "123:456",
+				User: "123456:456789",
 			},
 		},
 		state: &ContainerState{
@@ -26,14 +26,14 @@ func TestGenerateUserPasswdEntry(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	assert.Equal(t, user, "123:*:123:456:container user:/:/bin/sh\n")
+	assert.Equal(t, user, "123456:*:123456:456789:container user:/:/bin/sh\n")
 
-	c.config.User = "567"
+	c.config.User = "567890"
 	user, err = c.generateUserPasswdEntry(0)
 	if err != nil {
 		t.Fatal(err)
 	}
-	assert.Equal(t, user, "567:*:567:0:container user:/:/bin/sh\n")
+	assert.Equal(t, user, "567890:*:567890:0:container user:/:/bin/sh\n")
 }
 
 func TestGenerateUserGroupEntry(t *testing.T) {
@@ -41,7 +41,7 @@ func TestGenerateUserGroupEntry(t *testing.T) {
 		config: &ContainerConfig{
 			Spec: &spec.Spec{},
 			ContainerSecurityConfig: ContainerSecurityConfig{
-				User: "123:456",
+				User: "123456:456789",
 			},
 		},
 		state: &ContainerState{
@@ -52,12 +52,12 @@ func TestGenerateUserGroupEntry(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	assert.Equal(t, group, "456:x:456:123\n")
+	assert.Equal(t, group, "456789:x:456789:123456\n")
 
-	c.config.User = "567"
+	c.config.User = "567890"
 	group, err = c.generateUserGroupEntry(0)
 	if err != nil {
 		t.Fatal(err)
 	}
-	assert.Equal(t, group, "567:x:567:567\n")
+	assert.Equal(t, group, "567890:x:567890:567890\n")
 }

